.github/workflows/release.yml

```diff
@@ -7,7 +7,6 @@ on:
       - '[0-9]+.[0-9]+.x'
 jobs:
   release:
-    if: "!contains(github.event.head_commit.message, 'skip ci')"
```

Comment:
This isn't needed cause GitHub added native support for it right?

Reply:
Yeah I've just been removing it as we go.